test/CardRender.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  CardRender.swift
+//  test
+//
+//  Created by apple on 07/06/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class cardRender
+{
+    public func getSimbol(for card: Card) -> String
+    {
```

Comment:
I recommend adding opening braces on the same line with the block/method body. We have this convention on all other projects.